runtime/src/installed_scheduler_pool.rs

```diff
@@ -445,18 +446,18 @@ impl BankWithScheduler {
     /// Calling this will panic if the installed scheduler is Unavailable (the bank is
     /// wait_for_termination()-ed or the unified scheduler is disabled in the first place).
     // 'a is needed; anonymous_lifetime_in_impl_trait isn't stabilized yet...
```

Comment:
could you remove the last line of this comment, because `'a` gone: ` // 'a is needed; anonymous_lifetime_in_impl_trait isn't stabilized yet...`

Reply:
2b58020de0087ec9b10521a17d2f5a9787bc2e49